src/GameClass.py

```diff
@@ -0,0 +1,64 @@
+from TicTacToeLoop import takeTurns
+
+class GameState:
+    def __init__ (self):
+        self.board = [["1", "2","3"],
+         ["4", "5", "6"],
+         ["7", "8", "9"]]
+
+        self.possibleInputs = ["1", "2","3",
+         "4", "5", "6",
+         "7", "8", "9"]
```

Comment:
^ this would also be something to think about if/when you tackled the variable board size feature. Think about how it might be tough like this, hardcoded as a specific data structure/shape 🤔

Reply:
_createGameboard() and _createInputArray() have been made to initialize a game board and array. Both take in a single argument gameWidth. The argument assumes all games will have an even board creating. A gameWidth by gameWidth matrix is created, as well as an input array of the integers up to the gameWidth squared.  GameState's init() has been cleared of initialized board and inputArray.